internal/backend/sessions/sessioncalls/calls.go

```diff
@@ -76,11 +78,12 @@ func New(z *zap.Logger, config Config, svcs *sessionsvcs.Svcs) Calls {
 	opts.DisableWorkflowWorker = true // these workers serve only activities.
 
 	cs := calls{
-		z:             z,
-		config:        config,
-		svcs:          svcs,
-		generalWorker: worker.New(svcs.Temporal, taskQueueName, opts),
-		uniqueWorker:  worker.New(svcs.Temporal, uniqueWorkerCallTaskQueueName(), opts),
+		z:                    z,
+		config:               config,
+		svcs:                 svcs,
+		generalWorker:        worker.New(svcs.Temporal, taskQueueName, opts),
+		uniqueWorker:         worker.New(svcs.Temporal, uniqueWorkerCallTaskQueueName(), opts),
+		executorsForSessions: make(map[sdktypes.SessionID]*sdkexecutor.Executors, 16),
```

Comment:
Why 16?

Reply:
no real reason. just to prep the map with some capacity.